src/scheduler/queue_info.c

```diff
@@ -381,6 +411,13 @@ query_queue_info(status *policy, struct batch_status *queue, server_info *sinfo)
 			if(qinfo->backfill_depth > 0)
 				policy->backfill = 1;
 		}
+		else if(!strcmp(attrp->name, ATTR_partition)) {
+			qinfo->partition = string_dup(attrp->value);
+			if (qinfo->partition == NULL) {
+				log_err(errno, "query_queue_info", MEM_ERR_MSG);
```

Comment:
Use schdlog() and \_\_func\_\_

Reply:
Have changed to use \__func__. And please look into my above response for the rest of the comment.

Reply:
Thanks for letting me know this. I have made the changes accordingly. It seems there are several places where still log_err() is used with MEM_ERR_MSG which lead to my initial confusion.